python/paddle/fluid/dygraph/checkpoint.py

```diff
@@ -16,12 +16,155 @@
 
 import os
 import collections
-from ..framework import Variable, default_main_program
+from ..framework import Variable, default_main_program, in_dygraph_mode, dygraph_only
 import pickle
 from . import learning_rate_scheduler
 import warnings
+from .. import core
 
-__all__ = ['save_persistables', 'load_persistables']
+__all__ = [
+    'save_persistables', 'load_persistables', 'save_parameter',
```

Comment:
and better not call it save_parameter, since fluid.save_parameter may cause confusion